_docs/style/text-conventions/external-internal-links.md

```diff
@@ -0,0 +1,52 @@
+---
+title: External and Internal Link Guidelines
+layout: docs
+parent: Text Conventions
+category: Style
+usage: |
+  Use these guidelines when you construct links to external or internal content.
+resource: false
+status: complete
+last-modified: 2019-03-18
+---
+
+{% include toc.html %}
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-6-md hxSpan-6-lg" %}
+
+## General guidelines
+
+Use the [links style]({{site.baseurl}}/style/typography.html#link) for control panel page links. Use internal links for navigation. Consider the user's context and information needs when you decide how links should behave.
+
+We recommend that the link does not open a tab unless it is warranted. Opening a new tab unnecessarily compromises user experience. The following examples might help you decide when to open a tab:
+
+- If a link provides supplemental information that is useful to the user in completing the current workflow, and the user expects to return to the current tab, consider opening the content in another tab. For example, open help documentation in another tab.
+- If the user wants to change contexts, then open the content in the current tab. For example, when the user wants to navigate to a different area of the application, open the link in the current tab.
```

Comment:
Maybe Add: remember to use an external link icon if the user will navigate to a different domain even if it is within the same tab.

Reply:
Done.